SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -174,6 +176,8 @@ void CFlowOutput::SetAnalyzeSurface(const CSolver* const*solver, const CGeometry
   vector<su2double> Surface_MassFlow_Abs      (nMarker,0.0);
   su2activematrix Surface_Species(nMarker, nSpecies);
   Surface_Species = su2double(0.0);
+  su2activematrix Surface_Scalars(nMarker, nScalars);
+  Surface_Scalars = su2double(0.0);
```

Comment:
dead code... I'm trying to make this quick but I have to say it would be a lot faster if a little more attention was given, please review ALL your changes in light of the comments made, if each time you only address the one line where someone comments... You also teach so I'm sure you know this model doesn't work.

Reply:
Heh, yes, I hate it when students do that. Thanks for keeping me sharp. I removed all the old surface_scalar output throughout the code.